<launch>
    <node pkg="freshr" type="compatibility_scale.py" name="compatibility" output="screen" ns="compatibility">

        <param name="desired_keypoints"  value="[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16]"/>
        <param name="safety"  value="System Starting"/>
        <param name="conf_thr" type="double" value="0.9" />
        <param name="dmax" type="double" value="5.0" />
        <param name="dmin" type="double" value="0.5" />
        <param name="vmax" type="double" value="2.0" />
        <param name="amax" type="double" value="0.7" />

    </node>
</launch>


